src/main/java/net/minestom/server/event/player/EntitySwingHandEvent.java

```diff
@@ -0,0 +1,54 @@
+package net.minestom.server.event.player;
+
+import net.minestom.server.entity.LivingEntity;
+import net.minestom.server.event.trait.CancellableEvent;
+import net.minestom.server.event.trait.EntityInstanceEvent;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Called when a living entity swings its hand.
+ *
+ * @see LivingEntity#swingHand(LivingEntity.Hand)
+ */
+public final class EntitySwingHandEvent implements EntityInstanceEvent, CancellableEvent {
```

Comment:
Should not be in the `event.player` package.

Though more generally, I am unclear on the need for this to be an event. Whats the use case?

Reply:
I will move the event to another package. But the use case of this is to cancel behavior, such as sending a hand swing packet to viewers. Another use case might be my minestom combat implementation, we need this event to reset the attack strength for a player.